src/DssSpell.t.sol

```diff
@@ -189,19 +175,18 @@ contract DssSpellTest is DssSpellTestBase {
         //assertEq(OsmAbstract(0xF15993A5C5BE496b8e1c9657Fd2233b579Cd3Bc6).wards(ORACLE_WALLET01), 1);
     }
 
-    // NOTE: not for goerli
-    function testRemoveChainlogValues() private { // make private to disable
+    function testRemoveChainlogValues() public { // make private to disable
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // try chainLog.getAddress("RWA015_A_OUTPUT_CONDUIT_LEGACY") {
-        //     assertTrue(false);
-        // } catch Error(string memory errmsg) {
-        //     assertTrue(cmpStr(errmsg, "dss-chain-log/invalid-key"));
-        // } catch {
-        //     assertTrue(false);
-        // }
+        try chainLog.getAddress("SUPROXY_SPARK") {
```

Comment:
Is there any reason why we are using `fail` instead of `assertTrue` here? 

Pattern from mainnet

```
try chainLog.getAddress("SUBPROXY_SPARK") {
            assertTrue(false);
        } catch Error(string memory errmsg) {
            assertTrue(_cmpStr(errmsg, "dss-chain-log/invalid-key"));
        } catch {
            assertTrue(false);
        }
```

The great irony of this test type is that 'invalid key' also applies when we give it the wrong key ;)

Such as here:

```suggestion
        try chainLog.getAddress("SUBPROXY_SPARK") {
```

Other than this we can use the `assertTrue` pattern in line with the other stuff unless `fail` provides some advantage.

Reply:
`fail` allows you to specify a specific error message.
`assertTrue(false)` is merely a workaround because `fail` is a `forge-std` function, not a `ds-test` one.
I can revert it back if you don't think this is an improvement or if it should be done on a later time.

Reply:
Fixed the typo BTW.